packages/delivery-service/src/validation/notification/regexValidation.ts

```diff
@@ -17,7 +17,7 @@ export const checkRegexPattern = (
                 .match(EMAIL_REGEX);
             return patternCheck;
         case NotificationChannelType.PUSH:
-            return typeof recipientValue !== 'string' ? true : false;
+            return recipientValue.length > 300 ? true : false;
```

Comment:
what is this line doing ?

Reply:
Th subscription value that comes from frontend is of type string and genrally its value is of minimum length 300, so that validation is set.

Reply:
we cant use the length of the string to determine the notification channel that is just to opaque. Why do we not check if the string contains all the information needed to parse the channel

Reply:
Okay, I will make the changes in validation